mobile/android/android-emulator-services.ts

```diff
@@ -365,7 +365,11 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 		var platform = target.replace('android-', '');
 		var platformNumber = +platform;
 		if (isNaN(platformNumber)) {
-			if (platform === "L") {
+			// this might be a google image
+			platform = target.split(":")[2];
+			if (platform) {
+				platformNumber = +platform;
```

Comment:
Well, if is a strange image, the code won't work anyway. Right now it returns undefined and the maps() after that does not find a suitable image and we give a message "no image found". I can make it throw an exception but  this ultimately would give the same message to the user, right?